packages/bff/src/services/catalogService.ts

```diff
@@ -570,5 +572,41 @@ export function catalogServiceBuilder(
 
       return toBffCatalogApiEserviceRiskAnalysis(riskAnalysis);
     },
+    deleteEServiceDocumentById: async (
+      eServiceId: EServiceId,
+      descriptorId: DescriptorId,
+      documentId: EServiceDocumentId,
+      ctx: WithLogger<BffAppContext>
+    ): Promise<void> => {
+      await catalogProcessClient.deleteEServiceDocumentById(undefined, {
+        params: {
+          eServiceId,
+          descriptorId,
+          documentId,
+        },
+        headers: ctx.headers,
+      });
+    },
+    cloneEServiceByDescriptor: async (
+      eServiceId: EServiceId,
+      descriptorId: DescriptorId,
+      ctx: WithLogger<BffAppContext>
+    ): Promise<bffApi.CreatedEServiceDescriptor> => {
+      const eService = await catalogProcessClient.cloneEServiceByDescriptor(
+        undefined,
+        {
+          params: {
+            eServiceId,
+            descriptorId,
+          },
+          headers: ctx.headers,
+        }
+      );
+      const eServiceDescriptorId = eService.descriptors.at(0)?.id;
+      if (!eServiceDescriptorId) {
+        throw noDescriptorInEservice(eService.id);
+      }
```

Comment:
I am not sure about how meaningful is this error naming (I have seen that it's the same in the previous implementation).
Maybe we could rename it to something more specific for this use case, like `descriptorHasNotBeenCloned`.
cc @galales

Reply:
so far error names referenced to the cause of the error (eg. invalid state) rather than the consequences (eg. descriptor not published).
We should keep the current approach

Reply:
I got the point after offline discussion.
No strong opinions, I may suggest something like `missingDescriptorInClonedEservice`

Reply:
20cd168989a082ca11c3cf77b5f0170a868763b3